core/metacling/src/TClingCallFunc.cxx

```diff
@@ -1997,7 +1997,7 @@ TClingCallFunc::InitRetAndExecNoCtor(clang::QualType QT, cling::Value &ret) {
       return [this](void* address, cling::Value& ret) { exec(address, &ret.getPtr()); };
    } else if (QT->isMemberPointerType()) {
       const MemberPointerType *MPT = QT->getAs<MemberPointerType>();
-      if (MPT->isMemberDataPointer()) {
+      if (MPT && MPT->isMemberDataPointer()) {
```

Comment:
It probably doesn't matter but this new test is redundant with the test 2 line above.